### PR TITLE
Tweak Compiler Optimization Settings

### PR DESCRIPTION
**Pull request by LukeUsher**
This should bring a slight to moderate performance increase.

### CODE REVIEW

File: build/win32/Cxbx.vcxproj [hunk 1]
```diff
@@ -233,15 +233,17 @@
...
       <EnableEnhancedInstructionSet>StreamingSIMDExtensions2</EnableEnhancedInstructionSet>
       <MinimalRebuild>true</MinimalRebuild>
-      <ExceptionHandling>Sync</ExceptionHandling>
+      <ExceptionHandling>SyncCThrow</ExceptionHandling>
       <BufferSecurityCheck>false</BufferSecurityCheck>
       <ControlFlowGuard>false</ControlFlowGuard>
       <IntrinsicFunctions>true</IntrinsicFunctions>
       <FavorSizeOrSpeed>Speed</FavorSizeOrSpeed>
+      <FloatingPointModel>Precise</FloatingPointModel>
+      <WholeProgramOptimization>true</WholeProgramOptimization>
```
Comment (RadWolfie) [inline build/win32/Cxbx.vcxproj]: When this is set to true. Then how come "WholeProgramOptimization" with condition set to false base on your other commit here below?

File: build/win32/Cxbx.vcxproj [hunk 2]
```diff
@@ -438,6 +441,7 @@
       <PreprocessorDefinitions Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PreprocessorDefinitions Condition="'$(Configuration)|$(Platform)'=='Debug_Direct3D9|Win32'">%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <PreprocessorDefinitions Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <WholeProgramOptimization Condition="'$(Configuration)|$(Platform)'=='Release|Win32'">false</WholeProgramOptimization>
```
Comment (RadWolfie) [inline build/win32/Cxbx.vcxproj]: Okay, so CxbxKrnl.cpp source code cannot have optimization? By defining proper call such as stdcall, cdecl, etc. It shouldn't matter with optimization enabled.

I'm curious about this.

Comment (LukeUsher) [reply build/win32/Cxbx.vcxproj]: Applying LTCG to CxbxKrnl.cpp breaks the virtual memory placeholder: it moves the variable away from the xbe base address. Other optimisation are still applies there, just not LTCG

Comment (RadWolfie) [reply build/win32/Cxbx.vcxproj]: Ah, gotcha. Thanks for explanation. 😃

**Pull request merged**